conf/quincy/cephadm/tier-1_5node_cephadm_bootstrap.yaml

```diff
@@ -0,0 +1,48 @@
+globals:
```

Comment:
There are many services co-located... also with 6 OSD daemons... the memory usage would be over subscribed ... could we revisit the layout?

Also, please add some description about the purpose of the cluster layout.

Reply:
Thanks @psathyan

- Added description.
- For initial "quincy" Tier 1 test run we have added same "pacific" Tier1 after testing with ceph6.0 build
https://github.com/red-hat-storage/cephci/blob/master/suites/pacific/cephadm/tier-1_service_apply_spec.yaml
- We are planning to add more Tier 1 test suite in upcoming sprint with more test coverage after discussion.

Reply:
Okay.